models/DataObject/AbstractObject.php

```diff
@@ -1409,18 +1409,18 @@ public static function __callStatic($method, $arguments)
     protected static function makeList(array $listConfig, mixed $objectTypes): Listing
```

Comment:
I think this should be changed to `array`, as it'll probably break when trying to `array_diff()` it with some other type.

Reply:
Good point, thanks